application/models/item.py

```diff
@@ -12,6 +12,8 @@ class Item(db.Model):
     status = db.Column(db.String())
     category = db.Column(db.String())
     charity = db.Column(db.String())
+    charity_url = db.Column(db.String())
+    charity_score = db.Column(db.Integer())
```

Comment:
Possible conflict -- `charity_score` is listed here as an Integer

Reply:
Pushed new changes up - keeping charity_score as an int